agr_api/src/main/java/org/alliancegenome/api/service/GeneService.java

```diff
@@ -31,6 +32,8 @@
 
     @Inject
     private AlleleService alleleService;
+    @Inject
+    private CacheService cacheService;
```

Comment:
This `cacheService` should not be called directly. It should be called through the GeneCacheRepository in org.alliancegenome.cache.repository.

Reply:
Create a method in there GeneCacheRepository.getInteractions(id);